Added documentation (#20)
virelay/virelay: docs/source/getting-started/basic-usage.rst

On the far right of the ViRelAy UI, the eigenvalues of the embedding as well as the clusters of the currently selected clustering method can be seen. The eigenvalue plot shows the eigenvalues of the eigen decomposition of the currently selected embedding method in ascending order. Below the eigenvalue plot, a list of all clusters of the current clustering are displayed. These can be used to quickly select all sample points belonging to a cluster, by clicking the corresponding clustering.

Finally, at the top of the ViRelAy UI, below the project selection, the toolbox is situated. The toolbox allows the user to navigate the data of the current project. The toolbox can be seen in Figure 5.

.. figure:: ../../images/getting-started/basic-usage/toolbox.png
    :alt: Toolbox in the ViRelAy UI

>     :align: center

Author: The `sphinxcontrib-images` extension only works for images inserted using the `image` directive, but we are using the `figure` directive, because it allows us to have an image caption. I haven't found an alternative with support for the `figure` directive. We have 3 options:

1. Leave it as it is
2. Use the `image` directive and remove the image captions
3. Somehow crop the images to make them smaller

>     :align: center

Reviewer: Maybe split them after the last drop-down and line them up vertically?

>     :align: center

Author: I agree, this is the only image where it really is a problem, I am going to break it into smaller pieces and stack it vertically. Tell me if that is to your liking.

>     :align: center

Author: Done

===========
Basic Usage
===========

> If you already have a project file that you want to open, ViRelAy it can be started on the command line using the following command. If you do not have a project file, yet, you can read the :doc:`example-project` article, which guides you through the creation of a randomly generated project that you can use to familiarize yourself with the ViRelAy user interface.

Reviewer: ```suggestion
If you already have a project file that you want to open, ViRelAy can be started on the command line using the following command. If you do not have a project file yet, you can read the :doc:`example-project` article, which guides you through the creation of a randomly generated project that you can use to familiarize yourself with the ViRelAy user interface.
```

Author: Accepted

.. code-block:: console

    python -m virelay <project-file> [<project-file>, ...]

This will start the server at http://localhost:8080 and automatically open your default browser. Optionally, you can specify an alternative host and port using the ``--host`` and ``--port`` command line arguments.

> Please note, that starting ViRelAy using the command line interface is very easy, but it will start a rather slow development server. For improved load times or when running ViRelAy on a server, please run it using the WSGI HTTP server Gunicorn. For more information, please refer to :doc:`../user-guide/using-virelay-with-gunicorn`.

Reviewer: ```suggestion
Please note that while starting ViRelAy using the provided command line interface will start a rather slow development server. For improved load times or when running ViRelAy on a remote server, we recommend using the WSGI HTTP server Gunicorn. For more information, please refer to :doc:`../user-guide/using-virelay-with-gunicorn`.
```

Author: Accepted
